### DISCUSSION TITLE
Anydesk package in the Community repository

### DISCUSSION THREAD

**Discussion by schelpdesk** [chocolatey-community/chocolatey-packages#2422] [Q&A] [answered]
Hello, anyone can tell me if the anydesk 8.0.8 in the community repository is the one that is signed with the new certificate after their breach?

**Comment by TheCakeIsNaOH** [comment, marked as answer]
As per https://anydesk.com/en/changelog/windows, v8.0.8 is the version signed with the new certificate. I've confirmed that it was a recently generated (in the past few days) certificate, and that the checksums in the packages match the current installers. 

I've also unlisted all previous versions of the package, as it is unclear how far back the breach goes, according to their public statement.
https://anydesk.com/en/public-statement

The Anydesk packages are maintained at this repository, so open any further issues over there:
https://github.com/TheCakeIsNaOH/chocolatey-packages

**Reply by schelpdesk** [reply]
Thanks for the confirmation. I will test deploy this later.
